spec/functional/popup-spec.js

```diff
@@ -35,10 +35,12 @@ describe("Popup Events", () => {
         timeout: 1000,
       }
     );
-    const actual = $("p").getText();
-
-    expect($$("p").length).toBe(1);
-    expect(actual).toBe(expected);
+    $("p")
+      .getText()
+      .then((actual) => {
```

Comment:
fwiw if you wanted you could `await` it since `getText` returns a promise, but nothing wrong with this approach either just preference.